apps/dashboard/app/(app)/[workspaceSlug]/apis/[apiId]/_overview/components/table/components/log-details/components/log-section.tsx

```diff
@@ -1,123 +1,103 @@
-"use client";
-import { Card, CardContent, CopyButton } from "@unkey/ui";
+import { CopyButton } from "@unkey/ui";
 import React from "react";
 
+// Helper function to extract text from React elements
+const extractTextFromReactElement = (element: React.ReactNode): string => {
```

Comment:
why do we need this?

Reply:
In some cases we pass either `string` or `reactElement` and down the line probably someone requires type-safety. Lemme double check this one.